### PR TITLE
Add CATSLU results for XLS-R with mBART-50

### PR DESCRIPTION
**Pull request by akreal**

### CODE REVIEW

**Review by ftshijt — COMMENTED** [review]
Thanks for your contribution. It generally looks cool to me. A minor suggestion is to also add an entry in `egs2/README.md` for this recipe.

**Comment by akreal** [comment]
Hi @ftshijt ! Thanks for your review. I've added an entry in `egs2/README.md`.

File: egs2/catslu_entity/asr1/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- ASR config: [conf/tuning/train_asr_branchformer_xlsr_mbart.yaml](conf/tuning/train_asr_branchformer_xlsr_mbart.yaml)
+
+## Environments
+- date: `Wed Nov 23 11:38:33 CET 2022`
+- python version: `3.9.15 (main, Nov  9 2022, 00:00:00)  [GCC 11.3.1 20220421 (Red Hat 11.3.1-3)]`
+- espnet version: `espnet 202209`
+- pytorch version: `pytorch 1.12.1+cu116`
+- Git hash: `24cfde7f3fa3a8e4abab56c8238f7ab45e757507`
+- Commit date: `Tue Nov 22 16:23:07 2022 +0100`
+
```
Comment (sw005320) [inline egs2/catslu_entity/asr1/README.md:12]: Can you add a model checkpoint?

**Comment by sw005320** [comment]
Thanks a lot!
It looks good to me!

**Pull request merged**